lib/driver/module.cc

```diff
@@ -252,12 +286,21 @@ std::string cu_module::compile_llvm_module(llvm::Module* module, driver::device*
   int ptx = vptx(version);
   int ptx_major = ptx / 10;
   int ptx_minor = ptx % 10;
+#endif
   // create
   llvm::SmallVector<char, 0> buffer;
+#ifdef __HIP_PLATFORM_AMD__
+  std::string triple = "amdgcn-amd-amdhsa";
+  std::string proc = "gfx908"; //TODO: add code to QUERY GPU to make sure it works
```

Comment:
This hardcoding will need to be fixed in future PRs. As discussed with @micmelesse earlier, if we don't have a HIP API to provide us the gfx arch, we will need to request that from HIP team in a JIRA. cc @sunway513

Reply:
I add a function to grep for this info using rocminfo.